styleguide/components/21-atoms/link/_link.scss

```diff
@@ -6,37 +6,62 @@ a {
   text-decoration: underline;
   cursor: pointer;
 
+  &:hover {
+    @include theme('background-color', 'color-primary--lighten-4', 'link-background-hover');
+    @include theme('color', 'color-primary--darken-2', 'link-color-hover');
+  }
+
   &:focus {
     outline: thin dotted;
```

Comment:
Isn't this the default?
Was this hidden at some point?

Reply:
The default is the standard browser specific outline... It's rather ugly though...

Reply:
Yes, but it's what users recognize.
If we deviate, we should style it properly and deviate everywhere.